src/Server.php

```diff
@@ -1007,8 +1007,6 @@ public function __construct(
 			)));
 			$this->logger->info($this->language->translate(KnownTranslationFactory::pocketmine_server_license($this->getName())));
 
-			DefaultPermissions::registerCorePermissions();
-
```

Comment:
Only just caught this, but this could be problematic if plugins register permissions that conflict with the built-in ones. `RegistryTrait` will only register the permissions when the first permission is accessed. Sure, technically `SimpleCommandMap` should trigger it but I don't feel happy about relying on that.

Reply:
Protecting the namespace `pocketmine.` for registering permission could be a solution. I don't really like it but I don't see another way to do it atm

Reply:
Not sure how that would work without pre-registering the permissions somehow. Maybe `PermissionManager::make()` should do it similar to how we setup states for `RuntimeBlockStateRegistry`.

Reply:
I thinks plugins don't must remove/override a default permission

Reply:
It's not good enough to just say "pls don't do this". It shouldn't be possible to begin with.

Reply:
Yes, it's must be add in other PR. Maybe, i can do this if it's necessary for submit this pr

Reply:
It's a problem that this PR is introducing. Because of the use of `RegistryTrait`, the permissions will be lazily loaded, which could easily end up being after plugins load if someone makes a wrong refactor. Relying on lazy loading for something like this is risky.

The old code's explicit initialization made sure this wouldn't be a problem, but now it's gone.

Reply:
Ok so i need to re register `DefaultPermissions` after loadings of plugin ? it's that ?

Reply:
No, it needs to be initialized explicitly here, as it was originally.